### PR TITLE
Update TFM to netstandard2.0

### PR DESCRIPTION
**Pull request by jbagga**

### CODE REVIEW

File: build/common.props [hunk 1]
```diff
@@ -17,7 +17,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework' AND '$(OutputType)'=='library'">
-    <PackageReference Include="NETStandard.Library" Version="$(BundledNETStandardPackageVersion)" />
+    <PackageReference Include="NETStandard.Library.NETFramework" Version="$(NETStandardLibraryNETFrameworkVersion)" />
```
Comment (pranavkm) [inline build/common.props]: `PrivateAssets="All"`

File: build/common.props [hunk 2]
```diff
@@ -17,7 +17,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework' AND '$(OutputType)'=='library'">
```
Comment (pranavkm) [inline build/common.props]: `<ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">`

**Comment by jbagga** [comment]
@pranavkm Updated. Appveyor should pass after IISIntegration changes are in

**Comment by pranavkm** [comment]
Could you wait until @JunTaoLuo merges in IISIntegration and the build passes? Looks :shipit: otherwise

**Comment by JunTaoLuo** [comment]
This should now be unblocked.

**Comment by jbagga** [comment]
I'll wait for Coherence to pass. Right now it doesn't build locally. Thanks!

**Pull request merged**